src/app/fyle/merge-expense/merge-expense.page.ts

```
@@ -633,7 +633,7 @@ export class MergeExpensePage implements OnInit, AfterViewChecked {
       txn_dt: this.genericFieldsFormValues.dateOfSpend,
       receipt_ids: this.selectedReceiptsId,
       custom_properties: [
-        ...this.customInputsFormValues.fields,
+        ...this.customInputsFormValues?.fields,
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)